### DISCUSSION TITLE
Cancel subscription

### DISCUSSION THREAD

**Discussion by IndrajeethY** [community/community#65780] [New to GitHub]
Hey there i was using github student pack. And i just saw now that my account has been switched to pro (paid instance). I am not aware that my account was at paid instance. I am just getting mail to pay subscription can i cancel it out?

**Comment by arafajs** [comment]
1. Go to your GitHub account settings.
2. Click on "Billing & Plans".
3. Under "Subscriptions", click on "GitHub Student Developer Pack".
4. Click on "Cancel Subscription".

**Reply by kallol-brc** [reply]
There is no such option "GitHub Student Developer Pack" available under "Subscriptions".

**Reply by Zkaecehran** [reply]
no such place exists

**Reply by bizguide** [reply]
Outrageous bad directions nobody cares how screwed is this taking my money and not telling me how to stop it

**Reply by andyhu144** [reply]
git hub is so trash

**Reply by kleenkanteen** [reply]
In 2025 you have to click here in the overview page, took me a minute:
<img width="1237" alt="image" src="https://github.com/user-attachments/assets/9e712f78-b8fa-4212-a9d5-6fc17d24f2a6" />

**Reply by GPhelps3DArtAndCoding** [reply]
GitHub is shit!

**Comment by IndrajeethY** [comment]
I am billed for 100$ anyway to cancel it?

**Reply by ghost** [reply]
So I have to cancel through paypal or my card issuer?

**Comment by odah1** [comment]
I am trying to cancel my account/subscription how?

**Comment by comtsamsmith** [comment]
Me too .  I installed this and its cost a days work to restore dev envirmoent back to working.

**Comment by kiasar** [comment]
Has anyone found any solution!?

**Reply by Huangleizu** [reply]
I couldn't cancel my account

**Comment by mrrobot16** [comment]
Click the "Edit button" and then downgrade to free :)

<img width="1381" alt="Screenshot 2024-04-30 at 9 32 41 AM" src="https://github.com/community/community/assets/761735/97a71486-9678-4d57-af8a-3e4fb4e8edd8">

**Reply by willianfalbo** [reply]
Not a good experience at all! It sounds like it was made to prevent subscription cancellations.

**Reply by joshua-chau** [reply]
![image](https://github.com/user-attachments/assets/43b13e4d-fad4-4987-b550-7dbe7d26df5c)

This is crazy, it still shows that I will be in the monthly plan even after I clicked continue to downgrade. That is ridiculous.

**Reply by locknard** [reply]
> I will be in the monthly plan even after I clicked continue to downgrade. That is ridiculous.

me too ridiculous

**Comment by bizguide** [comment]
Hello what does this email mean why is it so cryptic this is ridiculous!

**Comment by NickUrs** [comment]
![Screenshot 2024-06-14 201807](https://github.com/community/community/assets/45332833/b349716a-1d2b-4e79-8018-31e4d30b0fb6)
here the solution
